doc/userguide/devguide/codebase/index.rst

```diff
@@ -10,3 +10,4 @@ Working with the Codebase
    testing
    unittests-c
    unittests-rust
+   installation-from-git
```

Comment:
When I look at the table of contents for the Devguide, I think this section should go before `code-style` to have a more semantical order.